code/ui/QuickShop.cs

```diff
@@ -47,20 +47,24 @@ public override void Tick()
 
         private class Header : Panel
         {
+            public Panel PriceHolder;
             public Label TitleLabel { get; set; }
+            public Label DollarSign { get; set; }
             public Label CreditsLabel { get; set; }
 
             public Header(Panel parent)
             {
                 Parent = parent;
 
                 TitleLabel = Add.Label("Shop", "title");
-                CreditsLabel = Add.Label("$ 0", "credits");
+                PriceHolder = Add.Panel("priceholder");
+                DollarSign = PriceHolder.Add.Label("$", "dollarsign");
+                CreditsLabel =  PriceHolder.Add.Label("0", "credits");
```

Comment:
Just a suggestion, up to you, do we want to continue with the `Label` postfix?

`PriceLabel`
`DollarSignLabel`
`CreditsLabel`

feel a bit better to me imo.

Reply:
Agree, Ill update